### ISSUE TITLE
New release ETA

### ISSUE THREAD

**Issue by SandroMachado** [PaoloRotolo/AppIntro#227]
Is it possible to provide a next release ETA, or release a beta with the new changes in master? There are a lot of improvements and bug fixes in master that are not available in the latest release. It would be really great to have at least a beta release from master that include the latest commits. 

Thanks for the great work.

**Comment by Andrew-Quebe** [comment]
@PaoloRotolo is in charge of releases, so I'll let him answer this issue. He's the one that releases everything on Maven.

**Comment by SandroMachado** [comment]
Thanks for the reply @Andrew-Quebe. 

I can easily make a PR to add support for [JitPack](https://jitpack.io/). Do you think the PR can be accepted? 

With JitPack I can use a commit hash (if the release is not available) to use the latest version. And it will never be required to publish to Maven or JCenter, it is just create the release on github and everyone can start to use the new release.

Please let me know WDYT.

**Comment by avluis** [comment]
@SandroMachado You can still use JitPack right now or am I not understanding this correctly.
Click on commits after adding the url and grab the SNAPSHOT:

```gradle
dependencies {
	        compile 'com.github.PaoloRotolo:AppIntro:-SNAPSHOT'
	}
```

**Comment by SandroMachado** [comment]
Usually we need to follow these [steps](https://jitpack.io/docs/ANDROID/):
```
To enable building on JitPack you need to add the android-maven plugin.

If using Gradle 2.4 or later:

1) In your root build.gradle:

buildscript { 
  dependencies {
    classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3' // Add this line
2) In your library/build.gradle add:

 apply plugin: 'com.github.dcendents.android-maven'  

 group='com.github.YourUsername'
3) Create a GitHub release or add a tag.
```

But looks like it is working without it, I will try.

**Comment by PaoloRotolo** [comment]
Sorry, I've been busy in this period.
I've just released AppIntro 4.0.0, changelog is here: https://github.com/PaoloRotolo/AppIntro/releases/tag/v4.0.0

**Comment by SandroMachado** [comment]
BTW, JitPack method works! Thanks
